grails-app/conf/Config.groovy

```diff
@@ -163,6 +163,14 @@ streamr.cps.maxConnTotal = 400
 // Maximum number of connections per route
 streamr.cps.maxConnPerRoute = 200
 
+// Data Union 2.0
+// https://new-staging.streamr.network/mainchain-rpc/http/
+// https://new-staging.streamr.network/sidechain-rpc/http/
+streamr.dataunion.mainnet.rpc.url = System.getProperty("streamr.dataunion.mainnet.rpc.url") ? System.getProperty("streamr.dataunion.mainnet.rpc.url") : "http://10.200.10.1:8545"
```

Comment:
@teogeb Is this duplicating main ethernet address configuration [1] or should DU2 configuration be separate from that? 

[1] https://github.com/streamr-dev/engine-and-editor/blob/f2674b713cc65362b074ee578b150e543e9e20cd/grails-app/conf/Config.groovy#L243

Reply:
To my understanding these two are generic, not specific to DU:
ENV DATAUNION_MAINNET_RPC_URL http://10.200.10.1:8545
ENV DATAUNION_SIDECHAIN_RPC_URL http://10.200.10.1:8546

And these two are specific to DU:
ENV DATAUNION_MAINNET_FACTORY_ADDRESS 0x5E959e5d5F3813bE5c6CeA996a286F734cc9593b
ENV DATAUNION_SIDECHAIN_FACTORY_ADDRESS 0x4081B7e107E59af8E82756F96C751174590989FE

If my understanding is correct, RPC urls should not have DATAUNION prefix.